src/test/java/ch/wisv/areafiftylan/integration/WebTournamentIntegrationTest.java

```diff
@@ -212,4 +234,67 @@ public void testDeleteAllTournamentsAsCommitteeMember() {
             body("message", is("All Tournaments deleted."));
         //@formatter:on
     }
+
+    @Test
+    public void testUpdateTournamentAsUser() {
+        User user = createUser();
+        Tournament tournament = createTournament(),
+                   updateTournament = updateTournament();
+        updateTournament.setId(tournament.getId());
+
+        given()
+            .header(getXAuthTokenHeaderForUser(user))
+            .body(updateTournament)
+            .contentType(ContentType.JSON)
+        .when()
+            .post(TOURNAMENT_ENDPOINT + tournament.getId())
+        .then()
+            .statusCode(HttpStatus.SC_FORBIDDEN);
+
+        assertEquals(1, tournamentRepository.findAll().size());
+        /* toString() is used to force it to check the values, rather than the objects (somehow this failed)
```

Comment:
Can't you simply fix this with a proper `equals()` method in `Tournament`?

Reply:
I could do that too, I was unsure if we want to write these ourselves. I'll do that :+1: